events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -0,0 +1,185 @@
+<html>
+    <head>
+        <title>Trivia</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="https://fonts.googleapis.com/css?family=Raleway:300,400,600,700" rel="stylesheet">
+        <link rel="stylesheet" href="trivia.css?065">
+    </head>
+    <body>
+        <h1>Trivia</h1>
+        <input type="button" class="gray" id="begin" value="Let's get started!">
+        <input type="button" class="gray" id="end" value="Game Over">
+        <h3>Questions</h3>
+        <div class="dropdown">
+            <input type="button" id="selectedType" onclick="toggleMenu('typeDropdown')" class="gray" value="Any Type   &#9660">
+            <ul class="dropdown-type">
+                <div id="typeDropdown" class="dropdown-items">
+                    <li>Any Type</li>
+                    <li>Multiple Choice</li>
+                    <li>True or False</li>
+                </div>
+            </ul>
+            <br>
+            <input type="button" id="selectedDifficulty" onclick="toggleMenu('difficultyDropdown')" class="gray" value="Any Difficulty   &#9660">
+            <ul class="dropdown-difficulty">
+                <div id="difficultyDropdown" class="dropdown-items">
+                    <li>Any Difficulty</li>
+                    <li>Easy</li>
+                    <li>Medium</li>
+                    <li>Hard</li>
+                </div>
+            </ul>
+            <br>
+            <input type="button" id="selectedCategory" onclick="toggleMenu('categoryDropdown')" class="gray" value="Any Category   &#9660">
+            <ul class="dropdown-category">
+                <div id="categoryDropdown" class="dropdown-items">
+                    <li value="9">General Knowledge</li>
```

Comment:
What is the purpose of these `value`s?

Reply:
Different categories of trivia? Or are you referring to something else?  Becky would really be the one to ask on this html.

Reply:
i'm referring to the numbers here. not worth changing, but it's strange to me to see these hard-coded values